tests/Avalonia.Controls.UnitTests/ItemsControlTests.cs

```diff
@@ -1182,7 +1182,6 @@ private static FuncControlTemplate CreateScrollViewerTemplate()
                         new ScrollContentPresenter
                         {
                             Name = "PART_ContentPresenter",
-                            [~ScrollContentPresenter.ContentProperty] = parent.GetObservable(ScrollViewer.ContentProperty).ToBinding(),
```

Comment:
While investigating this I noticed that some tests still have a content binding in their `ScrollViewer` templates which shouldn't be the case ever since #10803.